modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -0,0 +1,133 @@
+[[clients-opensuse]]
```

Comment:
Is it just me, or this new file is not linked at `nav-installation-guide.adoc`?

Reply:
Sorry, I mean `nav-client-configuration-guide.adoc`

Reply:
yes, it's now linked.  I hope it will work this way.